YIF_XUnitTests/Integration/YIF_Backend/Controllers/UsersControllerTests.cs

```diff
@@ -102,15 +103,17 @@ public async Task Send_ResetPassword_IfEmail_InCorrect(string email)
             string confirmPassword,
             string recaptcha)
         {
-            // Act
+            //Arrange
+            _userInputAttribute.SetUserIdForHttpContext();
             var content = new StringContent(JsonConvert.SerializeObject(new ChangePasswordApiModel
             {
-                UserId = _correctUserId,
                 OldPassword = oldPassword,
                 NewPassword = newPassword,
                 ConfirmNewPassword = confirmPassword,
                 RecaptchaToken = recaptcha
             }), Encoding.UTF8, "application/json");
+
```

Comment:
And there is a test, that has invalid UserId, but I guess it's not an appropriate test, cause you get userId, in your case, from context, so it will always be valid, but we should discuss it with team